### DISCUSSION TITLE
Increasing precision in a column

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1823] [Q&A] [answered]
I have the following setting in my `.visidatarc` file:

```
options.disp_float_fmt = '{:,.02f}'
```

This means that floats are displayed like 1.23, i.e. showing only two digits after the decimal dot.

Sometimes I'd like to say "This column has more sensitive data, I want to show a higher precision for it." I hope to do that in a way that involves less keypresses than typing an emtire custom `disp_float_fmt` pattern every time.

**Comment by saulpw** [comment, marked as answer]
Check out this person's plugin: https://github.com/andycraig/visidata-plugins/blob/main/precision.py

I'd like to bring it into VisiData core as a feature at some point.

**Reply by cool-RR** [reply]
Works well, thank you. I'd love it if it was included in VisiData. I changed the `precision-more` binding to `Alt+=`.

Both this binding and `Alt+-` don't work on Bash on Windows. They do work on WSL on Windows.
